src/tutorial/microbe_stage/MicrobeMovement.cs

```diff
@@ -84,28 +88,19 @@ public override void ApplyGUIState(MicrobeTutorialGUI gui)
             return false;
         }
 
-        protected override void OnProcess(TutorialState overallState, float delta)
+        [RunOnAxis(new[] { "g_move_forward", "g_move_backwards" }, new[] { -1.0f, 1.0f })]
+        [RunOnAxis(new[] { "g_move_left", "g_move_right" }, new[] { -1.0f, 1.0f })]
+        [RunOnAxisGroup]
+        public void OnMovement(float delta, float forwardBackwardMovement, float leftRightMovement)
```

Comment:
I didn't notice this before, but because this is in the *tutorial* and because of the way I wrote the tutorial system. This should be left as is, and not be converted to the new input system.